pyfakefs/fake_filesystem.py

```diff
@@ -4988,6 +4988,11 @@ def __iter__(self):
             self._raise('File is not open for reading')
         return self._io.__iter__()
 
+    def __next__(self):
+        if not self._read:
+            self._raise('File is not open for reading')
+        return self._io.__next__()
+
 
```

Comment:
Can you please add a line to make this work in Python 2? Something like:
```python
if sys.version_info < (3, ):
    next = __next__
```

Reply:
Addressed, also fix L4994 with `next(self._io)` for py2 compatiblity.

Reply:
Thanks!